src/main/java/gov/nasa/pds/tools/validate/rule/pds4/TableDataContentValidationRule.java

```diff
@@ -84,9 +94,40 @@
   *
   */
 public class TableDataContentValidationRule extends AbstractValidationRule {
-  
+  private static final Logger LOG = LoggerFactory.getLogger(TableDataContentValidationRule.class);
   private int PROGRESS_COUNTER = 0;
 
+  /** Used in checking for ASCII String related fields: should not have '+' in the format. */
+  private static final  Set<String> ASCII_STRING_TYPE_LIST = new HashSet<String>(); 
+  static {
+      ASCII_STRING_TYPE_LIST.add("ASCII_Short_String_Collapsed".toUpperCase());
+      ASCII_STRING_TYPE_LIST.add("ASCII_Short_String_Preserved".toUpperCase());
+      ASCII_STRING_TYPE_LIST.add("ASCII_String"                .toUpperCase());
+      ASCII_STRING_TYPE_LIST.add("ASCII_Text_Collapsed"        .toUpperCase());
+      ASCII_STRING_TYPE_LIST.add("ASCII_Text_Preserved"        .toUpperCase());
+  }
+
+  /** Used in checking ASCII Number related fields: should not have '-' in the format. */
+  private static final Set<String> ASCII_NUMBER_TYPE_LIST = new HashSet<String>();
+  static {
+      ASCII_NUMBER_TYPE_LIST.add("ASCII_NonNegative_Integer".toUpperCase());
+      ASCII_NUMBER_TYPE_LIST.add("ASCII_Numeric_Base16".toUpperCase());
+      ASCII_NUMBER_TYPE_LIST.add("ASCII_Numeric_Base2".toUpperCase());
+      ASCII_NUMBER_TYPE_LIST.add("ASCII_Numeric_Base8".toUpperCase());
+      ASCII_NUMBER_TYPE_LIST.add("ASCII_Real".toUpperCase());
+      ASCII_NUMBER_TYPE_LIST.add("ASCII_Integer".toUpperCase());
+  }
+
+  private static String PRODUCT_OBSERVATIONAL = "Product_Observational";
+  private static String TABLE_CHARACTER = "Product_Observational/File_Area_Observational/Table_Character";
+  private static String RECORD_CHARACTER = "Product_Observational/File_Area_Observational/Table_Character/Record_Character";
```

Comment:
@qchaupds could we use a shorter absolute XPath here instead of including this nesting? that way we do not have to update the code whenever a new product is introduced that uses it. e.g.

```
//Table_Character
//Table_Character/Record_Character
```

thoughts?